XamlAnimatedGif.Shared/Animator.cs

```diff
@@ -64,7 +64,7 @@ private Animator(Stream sourceStream, Uri sourceUri, GifDataStream metadata, Rep
             _timingManager = CreateTimingManager(metadata, repeatBehavior);
         }
 
-        internal static async Task<Animator> CreateAsync(Image image, Uri sourceUri, RepeatBehavior repeatBehavior = default(RepeatBehavior))
+        public static async Task<Animator> CreateAsync(Image image, Uri sourceUri, RepeatBehavior repeatBehavior = default(RepeatBehavior))
```

Comment:
Why do you need to make it public? I assume it's related to the brush test you added in the basic tests? I'd rather keep this method internal. I understand that it can be useful to make a brush with an animated GIF, but it could probably be done with a more specific API. Perhaps something like an `AnimatedBrush` helper class:

    var brush = AnimatedBrush.FromImage("animated.gif");

Anyway, it's not specific to Silverlight, so I'd rather handle it separately. But it's a useful feature, and I'd be glad to add it to the library. Could you please open another issue to discuss how it should be done?